eng/pipelines/common/global-build-job.yml

```diff
@@ -206,13 +206,23 @@ jobs:
           df -h
         displayName: Disk Usage before Build
 
-      # Install Wasm dependencies: emscripten, LLVM, NodeJS
-    - ${{ if and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.archType, 'wasm')) }}:
-      - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-emscripten.cmd $(Build.SourcesDirectory)\wasm-tools
+      # Install Wasm dependencies on Windows: emscripten, LLVM, NodeJS
+    - ${{ if and(eq(parameters.hostedOs, 'windows'), and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.archType, 'wasm'))) }}:
+      - script: pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-emscripten.ps1 $(Build.SourcesDirectory)/wasm-tools
         displayName: Install/activate emscripten
       - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.cmd $(Build.SourcesDirectory)\wasm-tools $(Build.SourcesDirectory) ${{ parameters.buildConfig }}
         displayName: Install/build LLVM
-      - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-nodejs.cmd $(Build.SourcesDirectory)\wasm-tools
+      - script: pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-nodejs.ps1 $(Build.SourcesDirectory)\wasm-tools
+        displayName: Install NodeJS
+
+      # Install Wasm dependencies on Linux: emscripten, LLVM, NodeJS.
+    - ${{ if and(eq(parameters.hostedOs, 'linux'), and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.archType, 'wasm'))) }}:
```

Comment:
Can we collapse the Windows/Linux into one section now (will require putting Emscripten on the same `pwsh` plan)?

Reply:
Yes, done